### PR TITLE
lab 2

### PR DESCRIPTION
**Pull request by snur206**

### CODE REVIEW

File: basiclibrary/lib/src/main/java/basiclibrary/Library.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    public boolean someLibraryMethod() {
+        return true;
+    }
+
+    // Rolling Dice
+    public int[] roll(int n) {
+        Random rand = new Random();
+        int[] diceRoll = new int[n];
+        for (int i = 0; i < n; i++) {
+            int randNum = rand.nextInt(6);
```
Comment (zork4cyber) [inline basiclibrary/lib/src/main/java/basiclibrary/Library.java:16]: This code would produce a result from 0 to 6. Then, adding one would change the range to 1-7. The goal was to roll a 6 sided die so refactor to fix!

File: basiclibrary/lib/src/main/java/basiclibrary/Library.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            diceRoll[i] = randNum + 1;
+        }
+        return diceRoll;
+    }
+
+    // Contains Duplicates
+    public boolean containsDuplicates(int[] array) {
```
Comment (zork4cyber) [inline basiclibrary/lib/src/main/java/basiclibrary/Library.java:23]: Good looking method!

File: basiclibrary/lib/src/test/java/basiclibrary/LibraryTest.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import static org.junit.jupiter.api.Assertions.*;
+
+class LibraryTest {
+    @Test void someLibraryMethodReturnsTrue() {
+        Library classUnderTest = new Library();
+        assertTrue(classUnderTest.someLibraryMethod(), "someLibraryMethod should return 'true'");
+    }
+
+    @Test void rollTest() {
+        Library sut = new Library();
```
Comment (reyjroliva) [inline basiclibrary/lib/src/test/java/basiclibrary/LibraryTest.java:16]: Arrange
`Library sut = new Library();
int[] testArray = new int[3];`

Act
`testArray = roll(3);`

Assert
`AssertTrue(testArray.length == 3)`
`AssertTrue(testArray[0] >= 1 && testArray[0] <= 6)`

File: basiclibrary/lib/src/test/java/basiclibrary/LibraryTest.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        System.out.println("Dice rolls: " + Arrays.toString(sut.roll(6)));
+    }
+    @Test void duplicateTest() {
+        Library sut = new Library();
+        int[] array = {2,4,6,2};
+        assertTrue(sut.containsDuplicates(array),"True or false?");
```
Comment (reyjroliva) [inline basiclibrary/lib/src/test/java/basiclibrary/LibraryTest.java:22]: update message "Array of {2, 4, 6, 2} asserts true, it contains a duplicate!"

**Pull request merged**